jwql/utils/utils.py

```diff
@@ -117,6 +125,88 @@ def _validate_config(config_file_dict):
         )
 
 
+def create_png_from_fits(filename, outdir):
+    """Create and save a png file of the provided file. The file
+    will be saved with the same filename as the input file, but
+    with fits replaced by png
+
+    Parameters
+    ----------
+    filename : str
+        Fits file to be opened and saved as a png
+
+    outdir : str
+        Output directory to save the png file to
+
+    Returns
+    -------
+    png_file : str
+        Name of the saved png file
+    """
+    if os.path.isfile(filename):
+        image = fits.getdata(filename)
+        ny, nx = image.shape
+        img_mn, img_med, img_dev = sigma_clipped_stats(image[4: ny - 4, 4: nx - 4])
+
+        plot = figure(tools='')
+        plot.x_range.range_padding = plot.y_range.range_padding = 0
+        plot.toolbar.logo = None
+        plot.toolbar_location = None
+        plot.min_border = 0
+        plot.xgrid.visible = False
+        plot.ygrid.visible = False
+
+        # Create the color mapper that will be used to scale the image
+        #mapper = LogColorMapper(palette='Viridis256', low=(img_med-5*img_dev) ,high=(img_med+5*img_dev))
+        mapper = LogColorMapper(palette='Greys256', low=(img_med-5*img_dev) ,high=(img_med+5*img_dev))
+
+        # Plot image
+        imgplot = plot.image(image=[image], x=0, y=0, dw=nx, dh=ny,
+                             color_mapper=mapper, level="image")
+
+        # Turn off the axes, in order to make embedding in another figure easier
+        plot.xaxis.visible = False
+        plot.yaxis.visible = False
+
+        # Save the plot in a png
+        output_filename = os.path.join(outdir, os.path.basename(filename).replace('fits','png'))
+        export_png(plot, filename=output_filename)
+        permissions.set_permissions(output_filename)
+        return output_filename
+    else:
+        return None
+
+
+def screenshot_as_png():
+    from bokeh.io.export import get_screenshot_as_png
```

Comment:
It turns out that this entire function I wrote, create_png_from_fits(), is not used anywhere. Should I get rid of it completely? Or is it potentially useful enough that it's worth keeping?

Reply:
It's so funny you said that because I thought this was `read_png`. I would say we should remove it if we don't use. Is there a chance you might want use it in the future?